cmd/protoc-gen-orm/main.go

```diff
@@ -1,7 +1,73 @@
 package main
 
-import "fmt"
+import (
+	"flag"
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/proto"
+	plugin "github.com/golang/protobuf/protoc-gen-go/plugin"
+	"github.com/gophersbd/ormpb/pkg/descriptor"
+	"github.com/gophersbd/ormpb/pkg/generator"
+	"github.com/spf13/pflag"
+)
 
 func main() {
-	fmt.Println("Hello I am going to be ORM. Please Code me up")
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+	pflag.Parse()
+	// Convinces goflags that we have called Parse() to avoid noisy logs.
+	flag.CommandLine.Parse([]string{})
+
+	Start()
+}
+
+func Start() {
```

Comment:
Write test for Start Methods, may be break it down to multiple methods and add tests for each. I do really care about having tests